src/Orleans.CodeGeneration.Build/Orleans.CodeGeneration.Build.csproj

```diff
@@ -36,16 +37,19 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="System.Runtime" Version="$(SystemRuntimeVersion)" PrivateAssets="All" />
-    <PackageReference Include="Newtonsoft.Json" Version="$(NewtonsoftJsonVersion)" PrivateAssets="All" />
-    <PackageReference Include="System.Runtime.Loader" Version="$(SystemRuntimeVersion)" PrivateAssets="All" />
-    <PackageReference Include="Microsoft.Extensions.DependencyModel" Version="$(MicrosoftExtensionsDependencyModelVersion)" PrivateAssets="All" />
-    <PackageReference Include="Microsoft.Extensions.Logging" Version="$(MicrosoftExtensionsLoggingVersion)" PrivateAssets="All" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="$(MicrosoftExtensionsLoggingVersion)" PrivateAssets="All" />
-    <PackageReference Include="Microsoft.Build.Framework" Version="$(MicrosoftBuildVersion)" PrivateAssets="All" />
-    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="$(MicrosoftBuildVersion)" PrivateAssets="All" />
+    <PackageReference Include="System.Runtime" Version="$(SystemRuntimeVersion)" />
+    <PackageReference Include="Newtonsoft.Json" Version="$(NewtonsoftJsonVersion)" />
+    <PackageReference Include="System.Runtime.Loader" Version="$(SystemRuntimeVersion)" />
+    <PackageReference Include="Microsoft.Extensions.DependencyModel" Version="$(MicrosoftExtensionsDependencyModelVersion)" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="$(MicrosoftExtensionsLoggingVersion)" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="$(MicrosoftExtensionsLoggingVersion)" />
+    <PackageReference Include="Microsoft.Build.Framework" Version="$(MicrosoftBuildVersion)" />
+    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="$(MicrosoftBuildVersion)" />
+  </ItemGroup>
+  <ItemGroup>
+    <!-- This must come after all other PackageReference elements -->
+    <PackageReference Update="@(PackageReference)" PrivateAssets="All" Publish="true" />
```

Comment:
I couldn't find much info on this. Is it an undocumented feature?

I did validate that it works though, both in command line and VS (in Windows, both .NET Framework and .NET Core)

Reply:
BTW, what I couldn't find information is on the Publish=true thing. I do understand PrivateAssets though

Reply:
I didn't find any docs on `Publish=true`, but I traced it back through to the targets file. See here: https://github.com/dotnet/sdk/blob/5d4aa5c712512861c5f0987f6695ca0868192aca/src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Publish.targets#L508